src/aeso_fcode_to_fate.erl

```diff
@@ -645,11 +652,11 @@ pp_op(loop) -> "LOOP";
 pp_op(I)    ->
     aeb_fate_pp:format_op(I, #{}).
 
-pp_arg(?i(I))      -> io_lib:format("~w", [I]);
-pp_arg({arg, N})   -> io_lib:format("arg~p", [N]);
-pp_arg({store, N}) -> io_lib:format("store~p", [N]);
-pp_arg({var, N})   -> io_lib:format("var~p", [N]);
-pp_arg(?a)         -> "a".
+pp_arg(?i(I))    -> io_lib:format("~w", [I]);
+pp_arg({arg, N}) -> io_lib:format("arg~p", [N]);
+pp_arg(?s(N))    -> io_lib:format("store~p", [-N]);
```

Comment:
I see, the stores are negative numbers and you want to print "store1" for `{store, -1}`

Reply:
No, that's just wrong. Look at the definition of `desugar_arg`:
```erlang
desugar_arg(?s(N)) -> {var, -N};
desugar_arg(A) -> A.
```